La ONU nunca morirá

fuihfuefuiewn opened the pull request:
This is connecting to the backend to show and recieve messages in real time.

AlexGarciaPrada committed:
Minimum call

AlexGarciaPrada committed:
Call implementation with the websocket service

AlexGarciaPrada committed:
Implementation with lamoara server

AlexGarciaPrada committed:
Allowed hosts update

AlexGarciaPrada committed:
Added env variables

Sa4dUs committed:
_

Sa4dUs committed:
Mute video on open

Sa4dUs committed:
_

Sa4dUs committed:
Pass subprotocols to websocket service constructor

Sa4dUs committed:
_

Sa4dUs committed:
Pass token fix

AlexGarciaPrada committed:
Folder Remade

Sa4dUs committed:
Add github workflows

fuihfuefuiewn committed:
Primera Cosa

Solo son las carpetas, pero que más da que lo suba

fuihfuefuiewn committed:
Primera Cosa

Solo son las carpetas, pero que más da que lo suba

fuihfuefuiewn committed:
Recieve re-hecho

Recieve= message history
Lo que pone en rojo es porque puse los nombres mal y usé componentes en vez de service...
Mañana más y mejor

fuihfuefuiewn committed:
Send re-hecho

Hay cosas que no compila, supongo que es para el fifis.
Así que bueno, pues eso

Sa4dUs commented on src/app/features/chat/message/send/send.service.ts:
```diff
@@ -0,0 +1,56 @@
+
+    message = new Subject<MessageFormat>();
+    open = new Subject<void>();
+    cerrar = new Subject<void>();//esta en español porque en inglés se me pone en rojo...(será nombre reservado)
```
it is: https://www.w3schools.com/js/js_reserved.asp

AlexGarciaPrada committed:
Request service and some samples

fuihfuefuiewn committed:
No subí esto.

Se quedó stasheado, es la corrección de message.component.
Lo que sucede es que sigo teniendo el mismo problema por como están montados los enlaces, se me la vida en saber donde tengo que meter el que me falta para probar lo que tengo

fuihfuefuiewn committed:
Link

El link está hecho, por fin...
Cualquiera diría...

fuihfuefuiewn committed:
Link

El link está hecho, por fin...
Cualquiera diría...

AlexGarciaPrada committed:
Some implementation

AlexGarciaPrada committed:
More implementations

AlexGarciaPrada committed:
Injections better than constructors

AlexGarciaPrada committed:
Update src/app/features/friendship/friend/friend.component.ts

Co-authored-by: Marcelo Domínguez <69964857+Sa4dUs@users.noreply.github.com>

AlexGarciaPrada committed:
Some details improved

Co-Authored-By: Marcelo Domínguez <69964857+Sa4dUs@users.noreply.github.com>
Co-Authored-By: Jorge Cimadevilla <109004232+Lamoara@users.noreply.github.com>

fuihfuefuiewn committed:
Merge branch 'main' into Unmimired_Chat

fuihfuefuiewn committed:
Bug

There existed a bug with routes due to different versions

fuihfuefuiewn committed:
Bug

Only God knows for how long this is happening

fuihfuefuiewn committed:
Bug

There existed a bug with routes due to different versions

fuihfuefuiewn committed:
Update websocket-service.ts

fuihfuefuiewn committed:
HTML of message component

This is the first version of the html of message.component

fuihfuefuiewn committed:
HTML of message component

Full vibecoded, if you don't like it cry :)

fuihfuefuiewn committed:
HTML of message component

This is the first version of the html of message.component

fuihfuefuiewn committed:
Recieve glow up

Was almost done, for some reason things like id where optional (for sure because it is easier to make it ccompile)
And there was a problem with routes

fuihfuefuiewn committed:
Update recieve.service.ts

fuihfuefuiewn committed:
Simplification of send

This was weird.
The created at of course is Mimir (don't even ask)

fuihfuefuiewn committed:
Update message.component.ts

fuihfuefuiewn committed:
Investigation of the real time problem
I am trying to find out the reason why it is not working

fuihfuefuiewn committed:
Chaos

fuihfuefuiewn committed:
Investigation of the real time problem
I am trying to find out the reason why it is not working

AlexGarciaPrada committed:
ws hotfix

AlexGarciaPrada committed:
ws hotfix

AlexGarciaPrada committed:
ws hotfix

fuihfuefuiewn committed:
Observable

And also the "plain text" is...polemic

Revert "Observable"

This reverts commit f9a16ef9acd75d47cb4fc82e0ea87862166121d3.

Advancements

No more 1011!!!
But still problems with messages reception

fuihfuefuiewn committed:
Observable

And also the "plain text" is...polemic

Revert "Observable"

This reverts commit f9a16ef9acd75d47cb4fc82e0ea87862166121d3.

Advancements

No more 1011!!!
But still problems with messages reception

fuihfuefuiewn committed:
The big bug

Only god knows how to print in the HTML the new messages

fuihfuefuiewn committed:
The big bug

Only god knows how to print in the HTML the new messages

fuihfuefuiewn committed:
Refactor Message.component

fuihfuefuiewn committed:
Refactor Message.component

fuihfuefuiewn committed:
ChannelId

ChannelId is no longer hardcoded
👍

fuihfuefuiewn committed:
ChannelId

ChannelId is no longer hardcoded
👍

fuihfuefuiewn committed:
lint

fuihfuefuiewn committed:
lint

fuihfuefuiewn committed:
Duplicated message

No more double message

fuihfuefuiewn committed:
Duplicated message

No more double message

fuihfuefuiewn committed:
Restore local versions after conflict
